package.json

```
@@ -1,15 +1,31 @@
 {
   "name": "rollup-plugin-obfuscator",
-  "version": "0.2.0",
+  "version": "0.2.1",
   "description": "The most powerful rollup plugin for javascript-obfuscator",
-  "main": "rollup-plugin-obfuscator.js",
+  "main": "dist/rollup-plugin-obfuscator.js",
```

```suggestion
  "main": "dist/rollup-plugin-obfuscator.js",
  "types": "dist/rollup-plugin-obfuscator.d.ts",
```

If this line isn't present, typescript and auto-completers will assume this is a JS project and you won't get typings when you use the plugin (as a dependency in another project I mean).

---

Didn't realized this, in my test it works fine as a dev dependency :P